src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+
+#### Cooperative
+Historically, due to the difficulty of surviving a round as a vox, they had a tendency to rely on each other to get by in a world that was just not built for them. When no one else "got it", vox players turned for support to those who did. (Or so I'm told). It is the position of this design that this cooperative aspect of them is something to be preserved, and it being species specific is not something that should either be encouraged or discouraged mechanically, leaving it to the players to decide.
+Obviously, "just designating" how a species is supposed to play like is an impossible, lofty goal, but nevertheless their features could be designed in a way to help with that
+This was the hardest pillar to phrase and will be the hardest to come up with ideas for. But if a feature works better with other players, does something for other players rather than you, or is a flaw that is more easily counteracted with external help rather than just having the right tool in your pocket, then that feature is Cooperative.
+
+
+## Specific Features and Ideas 
+(All subject to further review during the actual implementation, of course. Especially towards the end of the list. They are mostly just intended as a showcase of ideas to start out with)
+
+  - There should be some sort of warning or confirmation when selecting them, so players, especially new players, don't end up unwittingly picking a choice that will kill their character for the wrong click. 
+
+  - They breathe nitrogen, but oxygen hurts them badly (tentative number is, they crit in 2 minutes in normal air. Afaik this is actually more lenient than in Spess13)  - they will start with a large nitrogen tank and breathing mask equipped
+    - because taking their mask off is BAD, eating and drinking is challenging. They need a nitrogen room, and someone needs to have made that room
+    - possible alternative to a nitrogen room is the as-yet uncoded feature to hold their breath, taking asphyxiation damage instead of poison (at an even faster rate). They can regenerate that automatically (assuming they don't miscalculate and go crit)
+    - their emergency food and water could be replaced with injectable reagents
+  
+ - They can't be cloned
+ - They take longer to rot (2x? 3x?)
+ - They have 50% poison resistance (oxygen bypasses this. poison is generally not a very common damage type/not a very useful resistance to have, so I feel 50 is fine)
+ 
+ - They have their own blood type, and when that system is expanded, it should be incompatible with "normal blood"
+ - Possibly their brain, (it actually being a piece of tech), is automatically ready to be placed into a borg chassis
+
+  - They are small/unassuming/fragile.
+    - Possibly a dwarf-sized sprite? (when you sprite it)
```

Comment:
I think that they should NOT be sprited small (see: Borgs wearing any hat)

It would be far better if they were just downscaled to be smaller by nature.

Reply:
I'm told maintainers don't want sprite scaling because it looks bad

Reply:
> I'm told maintainers don't want sprite scaling because it looks bad

Didn't stop them with dwarves, a literal human reskin that adds zero gameplay value.

Reply:
Yeah but that's not the greatest example there is constantly talk about how they are a bad precedent, what if we removed them, they are just a debug species that got enabled for inscrutable reasons etc. Wouldn't use dwarf design as a role model

Reply:
Dwarfs are a really bad example of a species, I'm not sure why their even still in the game. Afaik they got added to add more species variety but they were originally intended as just a debug/test species. Some day they may be removed and replaced with a better designed species (Most likely Vox/Avalis or something smol :P)

We really need a better system to handle humanoid scaling. Until we get one though, scaling the sprite is probably the way to do it.